pkg/gadgets/filetop/tracer/bpf/filetop.bpf.c

```diff
@@ -0,0 +1,108 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
I am wondering if it would be possible to add like a git submodule to keep it in sync with upstream modification almost automatically?

Reply:
It could work but IMO the extra complexity of adding and maintaining a git submodule it's not worth it. Those files are not updated so often, so I think we can keep synchronizing them by hand in the meanwhile.

Reply:
In this case, I agree! So let's use the file itself in our repo.
By doing so, we also have the freedom to adapt it to suit our needs.

Reply:
Yes, that's also a good point, by having the file here we're able to modify it.